### ISSUE TITLE
Add a capability for synchronising query closes

### ISSUE THREAD

**Issue by csmith** [ShaneMcC/DFBnc#104]
It'd be nice if we had a protocol that allowed clients connected to bouncers to synchronise query closes.

DFBnc would need to advertise a capability and then relay new messages between clients that support it.

**Comment by ShaneMcC** [comment]
I'm thinking some kind of generic message-passing capability may be better.

**Comment by csmith** [comment]
Yeah, probably. I was thinking about mentioning unread statuses down the
line.

Maybe a CLIENTSYNC command that just gets proxied to other clients that
support it?

Then we can have `CLIENTSYNC CLOSE grebo`, `CLIENTSYNC READ #foo` etc.
